addon/components/paper-item.js

```diff
@@ -82,10 +82,12 @@ export default Component.extend(RippleMixin, ProxyMixin, {
         let el = component.$();
         set(this, 'mouseActive', false);
         el.on('mousedown', ()=> {
-          set(this, 'mouseActive', true);
-          run.later(()=> {
-            set(this, 'mouseActive', false);
-          }, 100);
+          if (!this.get('isDestroyed')) {
```

Comment:
I believe you will also want to check `isDestroying` and should probably be consistent with the rest of the file and use `get` not `this.get`